datajoint/base_relation.py

```diff
@@ -157,6 +157,39 @@ def _repr_helper(self):
         return "%s.%s()" % (self.__module__, self.__class__.__name__)
 
     # --------- SQL functionality --------- #
+    # TODO: finish this implementation
+    # @property
+    # def db_definition(self):
+    #     tdef = ...
+    #     if self.is_declared:
+    #         comment = self.table_comment
+    #         tdef = '# {comment}\n'.format(comment=comment) if comment else ''
+    #
+    #         #--------------------------
+    #         # TODO: remove this
+    #         from IPython import embed
+    #         embed()
+    #         exit()
+    #         #--------------------------
+    #
+    #         keys = ""
+    #         attributes = ""
+    #
+    #     return tdef
+
+    @property
+    def table_comment(self):
```

Comment:
This will soon be replaced with a single function that parses out the entire `CREATE TABLE` statement, not just the comment.